test/cmd/deployments.sh

```diff
@@ -132,6 +132,14 @@ os::cmd::expect_success 'oc delete hpa/test-deployment-config'
 echo "autoscale: ok"
 os::test::junit::declare_suite_end
 
+os::test::junit::declare_suite_start "cmd/deployments/setimage"
+os::cmd::expect_success 'oc create -f test/integration/testdata/test-deployment-config.yaml'
+os::cmd::expect_success 'oc set image dc/test-deployment-config ruby-helloworld=myshinynewimage'
+os::cmd::expect_success_and_text "oc get dc/test-deployment-config -o jsonpath='{.spec.template.spec.containers[0].image}'" "myshinynewimage"
+os::cmd::expect_success 'oc delete dc/test-deployment-config'
```

Comment:
Yeah, the DC is namespaced and will be deleted with the namespace cleanup after the suite is over -- unless you're adding the delete in there for a specific reason (other than the rest of the file doing it) we don't need the cleanup at all.